resources/config.test.edn

```diff
@@ -64,4 +64,5 @@
                                                                                                   :retry        {:count   [5 :int]
                                                                                                                  :enabled [true :bool]}}}}}
             :tracer               {:enabled         [true :bool]
-                                   :custom-provider ""}}}
+                                   :custom-provider ""}
+            :swagger              {:enabled false}}}
```

Comment:
instead of having a separate config, can we nest this under [:http-server :middlewares :swagger :enabled]?

Reply:
Sure.

Reply:
Done